flang/include/flang/Lower/CharacterExpr.h

```diff
@@ -159,15 +159,19 @@ class CharacterExprHelper {
   /// above instead of fixing it when needed.
   fir::ExtendedValue cleanUpCharacterExtendedValue(const fir::ExtendedValue &);
 
+  /// Create fir.char<k> singleton from \p code integer value.
+  mlir::Value createSingletonFromCode(mlir::Value code, int kind);
```

Comment:
I assume `kind` is the kind of singleton to be created?

Reply:
Yes, I changed the comment to `Create fir.char<kind> [...]` to make this clearer.